### PR TITLE
Tools: Add Presto panel source code

### PR DESCRIPTION
**Pull request by morpheus-14**

### CODE REVIEW

**Comment by dilipzotha** [comment]
Can you add .DS_Store to gitignore?

File: tools/presto-panel/JuspayPanel/manifest.json [hunk 1]
```diff
@@ -0,0 +1,176 @@
+{
+	"name": "JuspayPanel",
```
Comment (dilipzotha) [inline tools/presto-panel/JuspayPanel/manifest.json]: PrestoPanel

File: tools/presto-panel/JuspayPanel/manifest.json [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+	"authorEmail" : "___EMAIL___",
+	"homepage": "___HOMEPAGE___",
+	"identifier" : "PLUGIN_IDENTIFIER",
+	"compatibleVersion": 44,
+	"build": "",
+	"version": "PLUGIN_VERSION",
+    "appcast": "https://api.sketchpacks.com/v1/plugins/PLUGIN_IDENTIFIER/appcast",
+    "commands": [
+       {
+            "script" : "JuspayPanelUI.framework/JuspayPanelUI.js",
```
Comment (dilipzotha) [inline tools/presto-panel/JuspayPanel/manifest.json]: Can you change JuspayPanel to PrestoPanel everywhere?

File: tools/presto-panel/JuspayPanelUI/DUIPropsOverride.h [hunk 3]
```diff
@@ -0,0 +1,13 @@
+//
+//  DUIPropsOverride.h
+//  JuspayPanel
```
Comment (dilipzotha) [inline tools/presto-panel/JuspayPanelUI/DUIPropsOverride.h]: PrestoPanel

File: tools/presto-panel/JuspayPanel/manifest.json (hunk 1 above)
Comment (morpheus-14) [reply tools/presto-panel/JuspayPanel/manifest.json]: Alright

**Review by jronak — APPROVED** [review]
@dilipzotha LGTM

**Pull request merged**